src/client/Ruby/ms-rest-azure/lib/ms_rest_azure/active_directory_service_settings.rb

```diff
@@ -19,20 +19,44 @@ class ActiveDirectoryServiceSettings
     #
     # @return [ActiveDirectoryServiceSettings] settings required for authentication.
     def self.get_azure_settings
-      settings = ActiveDirectoryServiceSettings.new
-      settings.authentication_endpoint = 'https://login.windows.net/'
```

Comment:
This value seems to be different with the new settings:  'https://login.microsoftonline.com/', which one should it be?